### PR TITLE
Modified some code to account for pathway for new preprocessed data a…

### PR DESCRIPTION
**Pull request by noelpimentel**
@imranyousuf 
brain_activation.py is in its rough stage so it may change.
The last three codes must be run under data/. I downloaded the preprocessed data and put it under data/preprocessed_ds105 on my own laptop
There's a cleaner way of doing this and specifying pathways but idk how. Need help with it

We're using the condition files from the old data (i.e. ds105) and the and fmri data from the preprocessed data (i.e. preprocessed_ds105)

### CODE REVIEW

**Comment by noelpimentel** [comment]
@imranyousuf just added a txt file that is needed for the color of the activation maps. I don't know where it should be placed for organizing purposes

Also, brain_activation(subject, task_run, condition, TR = 2.5) where 
subject = 'sub001', 'sub002', 'sub003', etc.
task_run = 'task001_run001', 'task001_run002', etc
condition = 0,1,2,3,4,5,6,7,8, or 9

**Comment by imranyousuf** [comment]
ok. I will try to figure it out and arrange them

File: code/utils/cond_path.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def cond_path(subject, task_num, cond_num):
 		This is the path of the condition file that you choose.
 	"""
 
-	sub_path = os.path.realpath(subject)
+	sub_path = os.path.realpath('ds105/' + subject)
```
Comment (noelpimentel) [inline code/utils/cond_path.py:22]: added ds105 but should be able to find subjects/conditions regardless of pwd

File: code/utils/list_cond.py [hunk 2]
```diff
@@ -19,7 +19,7 @@ def list_every_cond(subject, task_num):
 
 	"""
 
-	sub_path = os.path.realpath(subject)
+	sub_path = os.path.realpath('ds105/' + subject)
```
Comment (noelpimentel) [inline code/utils/list_cond.py:22]: added ds105 but should be able to find subjects/conditions regardless of pwd

File: code/utils/path_bold.py [hunk 3]
```diff
@@ -17,7 +17,7 @@ def path_bold(subject, task_num):
 		this is the path of the bold.
 	"""
 
-	sub_path = os.path.realpath(subject)
-	bold_path = sub_path + '/BOLD' + '/' + task_num
+	sub_path = os.path.realpath('preprocessed_ds105/' + subject)
```
Comment (noelpimentel) [inline code/utils/path_bold.py:20]: added preporcessed_ds105 but should be able to find subjects/fmri data regardless of pwd

File: code/utils/brain_activation.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
+def brain_activation(subject, task_run, condition, TR = 2.5):
+	img = nib.load(path_bold(subject, task_run) + 'filtered_func_data_mni.nii.gz')
```
Comment (noelpimentel) [inline code/utils/brain_activation.py:11]: Again, should be cleaner. I don't know how to do so

File: code/utils/brain_activation.py [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+	beta_vols = np.zeros(vol_shape + (10,))
+	beta_vols[mask] = betas.T
+
+	mean_data[~mask] = np.nan
+	beta_vols[~mask] = np.nan
+
+	nice_cmap_values = np.loadtxt('actc.txt')
+	nice_cmap = colors.ListedColormap(nice_cmap_values, 'actc')
+
+	for i in range(mean_data.shape[-1]):
```
Comment (noelpimentel) [inline code/utils/brain_activation.py:55]: @imranyousuf exports 90 .pngs in pwd. Should be exported in a newly created folder based off condition label

**Comment by imranyousuf** [comment]
ok, I have to look into it more.

**Pull request merged**